src/TransitionSystems/location_id.rs

```diff
@@ -21,13 +23,64 @@ impl LocationID {
             "consistency: {}",
             string
         ))
+        .unwrap()
```

Comment:
Error?

Reply:
@t-lohse this is a problem for the "cause of failure" group. Sebastian said we could just leave them be for now.